pages/models.py

```diff
@@ -209,7 +209,6 @@ class RootPage(Page):
     # TODO: we are allowing this to be built as a child of the homepage. Not ideal.
     # Once the home page is released, use something to migrate homepage children to root page and remove this parent type.
     parent_page_types = ['wagtailcore.Page', 'pages.HomePage']
-    subpage_types = ['pages.FlexPage']  # which might also require allowing all pages to be children.
```

Comment:
it'll still be possible to create new flexpages after this right?

Reply:
yes, this is just saying that there are no restrictions on what can be a subpage of RootPage